catalog/sql/sql.go

```diff
@@ -133,8 +136,12 @@ type sqlIcebergTable struct {
 	CatalogName              string `bun:",pk"`
 	TableNamespace           string `bun:",pk"`
 	TableName                string `bun:",pk"`
+	IcebergType              string // "TABLE" or "VIEW"
 	MetadataLocation         sql.NullString
 	PreviousMetadataLocation sql.NullString
+	ViewSQL                  sql.NullString    // Only populated for views
+	SchemaJSON               sql.NullString    // Only populated for views
+	Properties               map[string]string `bun:"type:jsonb"` // Only populated for views
```

Comment:
You're still adding extra fields that shouldn't be there and don't get populated for views at all. We need to remove them so that we're complying with the way the JDBC catalog works.